picus/algorithms/dpvl.rkt

```diff
@@ -424,25 +424,22 @@
     (set! :extcnsts extcnsts)
     (set! :skip-query? skip-query?)
 
+    (define alt-xvec (list->set :alt-xlist))
```

Comment:
We should add a comment about the purpose of alt-xlist is doing. Also do you know why it is named xlist? I suppose it is because the variables are of the form x0 etc but it might be better to call it varlist or siglist